scripts/test_android_unit.sh

```diff
@@ -2,5 +2,8 @@
 
 ANDROID_PLUGIN_ROOT="$PROJECT_ROOT/unity_buy_android"
 
+# Run checkstyle before running tests
+"$ANDROID_PLUGIN_ROOT"/gradlew unity_buy_plugin:checkstyle -p "$ANDROID_PLUGIN_ROOT" || die "Failed to run checkstyle"
+
```

Comment:
Should I also be running the `check` task instead of `testDebug` for running tests? Will that run unit and instrument tests?